LuaSkin/LuaSkin/Skin.m

```diff
@@ -294,6 +338,15 @@ - (void)checkArgs:(int)firstArg, ... {
                 break;
             case LUA_TNUMBER:
                 lsType = LS_TNUMBER;
+
+                if ((spec & lsType) && (spec & LS_TINTEGER)) {
+                    int isInteger ;
+                    lua_tointegerx(_L, idx, &isInteger) ;
+                    if (!isInteger) {
```

Comment:
Ah... then I did a bad job in my description/documentation... I went with LS_TINTEGER being a modifier of LS_TNUMBER and not a stand-alone type... it's like LS_TOPTIONAL and means nothing by itself.  My bad for not being clearer...  Do you think it should be stand-alone?  My thinking was that if LS_TINTEGER truly was stand alone, then LS_TNUMBER | LS_TINTEGER has no meaning, since LS_TNUMBER would be sufficient without the alternative valid type... it seems more like the optional tag to me in that it modifies something but isn't a full definition itself.